source/tutorials/pd-mice.rst

```diff
@@ -432,7 +380,7 @@ One important consideration for diversity calculations is the rarefaction depth.
 Alpha diversity
 +++++++++++++++
 
-Alpha diversity asks whether the distribution of features within a sample differ between different conditions. The comparison makes no assumptions about the features that are shared between samples; two samples can have the same alpha diversity and not share any features. The rarefied alpha diversity produced by ``q2-diversity`` is a univariate, continuous value and can be tested using common non-parametric statistical tests.
+Alpha diversity asks whether the distribution of features within a sample (or groups of samples) differs between different conditions. The comparison makes no assumptions about the features that are shared between samples; two samples can have the same alpha diversity and not share any features. The rarefied alpha diversity produced by ``q2-diversity`` is a univariate, continuous value and can be tested using common non-parametric statistical tests.
```

Comment:
```suggestion
Alpha diversity asks whether the distribution of features within a sample (or groups of samples) differ between different conditions. The comparison makes no assumptions about the features that are shared between samples; two samples can have the same alpha diversity and not share any features. The rarefied alpha diversity produced by ``q2-diversity`` is a univariate, continuous value and can be tested using common non-parametric statistical tests.
```

Reply:
the distribution ... differs
not
the distribution ... differ

Reply:
Having verified, it still sounds weird, but eh. Grammar has never been my best subject